.buildkite/pipelines/intake.template.yml

```diff
@@ -63,6 +63,11 @@ steps:
           buildDirectory: /dev/shm/bk
         env:
           BWC_VERSION: "{{matrix.BWC_VERSION}}"
+  - label: bc-upgrade
+    command: ".buildkite/scripts/run-bc-upgrade-tests.sh"
+    agents:
```

Comment:
I would just remove the `agents:` configuration here altogether, and just use the default k8s agent. No need to use that eck-region agent, and doesn't look like the script needs 4GB of memory